vendor/bat-native-ads/src/bat/ads/internal/common/search_engine/search_engine_domain_extension_constants.cc

```diff
@@ -5,56 +5,56 @@
 
 #include "bat/ads/internal/common/search_engine/search_engine_domain_extension_constants.h"
 
-#include "base/no_destructor.h"
-
 namespace ads {
 
-const std::vector<std::string>& GetAmazonSearchEngineDomainExtensions() {
+base::span<const base::StringPiece> GetAmazonSearchEngineDomainExtensions() {
   // See https://www.amazon.com/gp/navigation-country/select-country.
-  static const base::NoDestructor<std::vector<std::string>> kExtensions(
-      {"ae",     "ca",     "cn", "co.jp", "co.uk", "com", "com.au",
-       "com.br", "com.mx", "de", "eg",    "es",    "fr",  "in",
-       "it",     "nl",     "pl", "sa",    "se",    "sp",  "tr"});
-  return *kExtensions;
+  static constexpr base::StringPiece kExtensions[] = {
+      "ae",     "ca",     "cn", "co.jp", "co.uk", "com", "com.au",
+      "com.br", "com.mx", "de", "eg",    "es",    "fr",  "in",
+      "it",     "nl",     "pl", "sa",    "se",    "sp",  "tr"};
+
+  return base::make_span(kExtensions);
```

Comment:
nit: `return kExtensions;` should also work fine in such constructions, no need to write `base::make_span`.